scripts/Minify-Scripts.ps1

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env powershell
```

Comment:
If these pwsh scripts are expected to run on x-plat, you'll need to make two changes:

* `#!/usr/bin/env pwsh`
* chmod +x

Reply:
There aren't expected to run x-plat, they're basically just convenience scripts for me to use when developing locally.